src/js/components/ImageMenuItem.vue

```diff
@@ -35,12 +34,10 @@ export default {
       open: false,
     }
   },
-  computed: {
-    ...mapGetters(['username', 'sessionID', 'userID'])
-  },
   methods: {
     fetchChildren() {
       this.children = []
+      // let children = []
```

Comment:
Is this needed (for something later) or should it be removed?

Reply:
Oops!  this is detritus from my struggles with unit testing and should be removed.  Thanks!  
See my answers to your questions regarding testing this function below.